.github/workflows/preview.yml

```diff
@@ -1,74 +1,48 @@
-          #
-          prettier_ignore: '*'
-          ### Ignore paths for prettier to not format those html files.
-          ### Useful if the file is exceptionally large, so formatting it takes a while.
-          ### Also useful if HTML compression is enabled for that file / formatting messes it up.
-          #
-          # jekyll_src: sample_site
-          ### If the jekyll website source is not in root, specify the directory. (in this case, sample_site)
-          ### By default, this is not required as the action searches for a _config.yml automatically.
-          #
-          # gem_src: sample_site
-          ### By default, this is not required as the action searches for a _config.yml automatically.
-          ### However, if there are multiple Gemfiles, the action may not be able to determine which to use.
-          ### In that case, specify the directory. (in this case, sample_site)
-          ###
-          ### If jekyll_src is set, the action would automatically choose the Gemfile in jekyll_src.
-          ### In that case this input may not be needed as well.
-          #
-          # key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-          # restore-keys: ${{ runner.os }}-gems-
-          ### In cases where you want to specify the cache key, enable the above 2 inputs
-          ### Follows the format here https://github.com/actions/cache
-          #
-          custom_opts: '--future'
-          ### If you need to specify any Jekyll build options, enable the above input
-          ### Flags accepted can be found here https://jekyllrb.com/docs/configuration/options/#build-command-options
-
-      - uses: afc163/surge-preview@v1
-        id: preview_step
-        with:
-          surge_token: ${{ secrets.SURGE_TOKEN }}
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          dist: _site
-          failOnError: 'true'
-          build: |
-            echo Deploying to surge.sh
-
-      - name: Get the preview_url
-        run: echo "url => ${{ steps.preview_step.outputs.preview_url }}"
+    - name: Download PR Artifact
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: ${{ github.event.workflow_run.workflow_id }}
+        workflow_conclusion: success
+        name: site
+    - name: Store PR id as variable
+      id: pr
+      run: |
+        echo "::set-output name=id::$(<pr-id.txt)"
+        rm -f pr-id.txt
+    - name: Publishing to surge for preview
+      id: deploy
+      run: npx surge ./ --domain https://quarkus-io-pr-${{  steps.pr.outputs.id }}-preview.surge.sh --token ${{ secrets.SURGE_TOKEN }}
+    - name: Update PR status comment on success
+      uses: actions-cool/maintain-one-comment@v1.2.1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        body: |
+          🎊 PR Preview has been successfully built and deployed to https://quarkus-io-pr-${{ steps.pr.outputs.id }}-preview.surge.sh
```

Comment:
It would be nice to show the commit used in the deployment: 

![image](https://user-images.githubusercontent.com/54133/155131418-4b1e83ee-786a-4c0a-b420-e5178e3a8312.png)

Reply:
@gastaldi yes could you add it in another PR?

Reply:
```suggestion
          🎊 PR Preview ${{ github.sha }} has been successfully built and deployed to https://quarkus-io-pr-${{ steps.pr.outputs.id }}-preview.surge.sh
```

Reply:
Aw I thought it would have been more complicated :)